core/src/luigi-config/communication.js

```diff
@@ -101,6 +102,9 @@ export const communication = {
         await setCluster(params?.kubeconfig?.['current-context']);
       }
     },
+    'busola.loadDefaultKubeconfigId': async () => {
+      await loadDefaultKubeconfigId();
```

Comment:
It is not an async function, we can get rid of `async`, `await` and `async` in `core/src/luigi-config/kubeconfig-id/loadKubeconfigById.js`.

Reply:
done